Cargo.toml

```diff
@@ -131,7 +131,7 @@ ofb = { version = "0.6.1", optional = true }
 
 # Dependencies used for non-WASM
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-dns-lookup = { version = "2.0.4", optional = true }
+dns-lookup = { version = "~2.0", optional = true }
```

Comment:
What was the issue here?

Reply:
I'm not sure. I retried and it now builds (on my laptop :)) Will revert.